packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -141,12 +140,9 @@ const convertStoreAndReturnPdfUrl = async ({
     // Defines page
     const page = await browser.newPage();
 
-    await page.setDefaultNavigationTimeout(puppeteerTimeoutInMillis);
-
+    page.setDefaultNavigationTimeout(puppeteerTimeoutInMillis);
```

Comment:
Whyd we remove the await here?

Reply:
the `setDefaultNavigationTimeout` is not an async function and the IDE was showing the warning that `await` wasn't doing anything there. 

This code isn't really relevant to my PR, but I'm just following the boy scout rule 🤓